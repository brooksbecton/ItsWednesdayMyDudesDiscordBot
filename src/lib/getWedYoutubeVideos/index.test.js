const getWedYoutubeVideos = require('./index');

describe('getWedYoutubeVideos', () => {
  test('fetches youtube data', async () => {
    const videoUrls = await getWedYoutubeVideos();
    expect(videoUrls.length).toMatchSnapshot();
  });
});
