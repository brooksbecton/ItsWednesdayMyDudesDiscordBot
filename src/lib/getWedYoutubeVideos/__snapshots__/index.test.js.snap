// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getWedYoutubeVideos fetches youtube data 1`] = `21`;
